templates/arista_eos_show_inventory.textfsm

```diff
@@ -11,26 +11,26 @@ Chassis
   ^\s+Model
   ^\s+-
   ^\s+HW
-  ^\s+${VID}\s+${SN}\s+\d+-\d+-\d+$$ -> Record
+  ^\s+${VID}\s+${SN}\s+\d+-\d+-\d+\s+\d+\.\d+\s*$$ -> Record
   ^\s+${NAME}?\s+${DESCR}$$
   ^\s*System.+(power supply|power-supply) -> Power_Supply
 
 
 Power_Supply
   ^\s+Slot
   ^\s+-
-  ^\s+${PORT}\s+${NAME}\s+${SN}$$ -> Record
```

Comment:
What if you just got rid of the end of line requirement?

e.g. removing the `$$` at the end of each of these?

Reply:
Thank you for your reply.

As you said, it was possible to just erase the $$ end of the line. 
e.g.   ^\s+${PORT}?\s+\d+?\s+${NAME}?\s+${SN} -> Record

Should I fix it?

Reply:
I believe so and revert the existing test files and then add your own example to make sure all use cases are covered.